apps/ui/components/reservation/EditStep1.tsx

```diff
@@ -141,7 +143,7 @@ const EditStep1 = ({
       }
 
       if (key === "ageGroup") {
-        const { minimum, maximum } = reservation[key];
+        const { minimum, maximum } = reservation[key] ?? {};
         return `${minimum} - ${maximum}`;
```

Comment:
I'd default to something else here. like "1 - " if neither. minimum = 1, maximum = ""